compiler/stable_mir/src/mir/body.rs

```diff
@@ -967,7 +967,7 @@ pub enum PointerCoercion {
 
 #[derive(Copy, Clone, Debug, Eq, PartialEq)]
 pub enum CastKind {
-    PointerExposeAddress,
+    PointerExpose,
```

Comment:
Please don't rename it here. Thanks

Reply:
Then where should we put the note to rename it in the future? It's quite annoying when we clean up naming but smir still uses old (and thus more confusing / less apt) names possibly indefinitely.

Reply:
Document the field instead.

Reply:
If it's just a comment it will get lost. It needs to carry at least some sort of marker so that when you eventually do follow suit and fix old confusing names in smir, you can reliably find them. Something one can grep for. Like `FIXME(smir-rename)` or so -- I don't care about the details, but I hope the smir team can come up with a plausible process and document it.

Reply:
Added a comment with `FIXME(smir-rename)`. And just fyi, #122935 made this kind of change but didn't get caught :smile:.